Microsoft.Toolkit/Diagnostics/ThrowHelper.ThrowExceptions.cs

```diff
@@ -13,22 +14,17 @@ namespace Microsoft.Toolkit.Diagnostics
     /// <summary>
     /// Helper methods to throw exceptions
     /// </summary>
-    internal static partial class ThrowHelper
+    public static partial class ThrowHelper
     {
         /// <summary>
-        /// Returns a formatted representation of the input value.
+        /// Throws a new <see cref="ArgumentException"/>.
         /// </summary>
-        /// <param name="obj">The input <see cref="object"/> to format.</param>
-        /// <returns>A formatted representation of <paramref name="obj"/> to display in error messages.</returns>
-        [Pure]
-        private static string ToAssertString(this object? obj)
+        /// <param name="message">The message to include in the exception.</param>
+        /// <exception cref="ArgumentException">Thrown with <paramref name="message"/>.</exception>
+        [DoesNotReturn]
+        public static void ThrowArgumentException(string message)
```

Comment:
Generally I would consider having an `Exception? inner = null` optional parameter. I've used it frequently enough that I consider it important

Reply:
Done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3346/commits/0172ae5d674dc5b25982a7b05f98ef1565e36482.